test/index.js

```diff
@@ -1,46 +1,63 @@
 const assert = require('assert')
 
 const dedupe = require('../build/dedupe')
-let Items = require('../index.js')
+let Items
 const decache = require('decache')
 
+const inits = []
+
+const wrapConstr = (opts) => {
+  const before = Date.now()
+  const items = new Items(opts)
+  const after = Date.now()
+  inits.push(after - before)
+  return items
+}
+
 beforeEach(() => {
   Items = require('../index')
 })
 afterEach(() => {
   decache('../index.js')
 })
+// after(() => {
+//   console.error(`Average init time: ${(inits.reduce((a, b) => a + b, 0) / inits.length).toFixed(0)}`)
+// })
```

Comment:
commented to allow debugging more later... cause test code, i don't care if it stays in, personally